addons/web/static/tests/views/form/form_view_tests.js

```diff
@@ -8445,7 +8445,7 @@ QUnit.module("Views", (hooks) => {
         await assertFormContainsNButtonsWithSizeClass(0, 2);
         await assertFormContainsNButtonsWithSizeClass(1, 3);
         await assertFormContainsNButtonsWithSizeClass(2, 4);
-        await assertFormContainsNButtonsWithSizeClass(3, 6);
+        await assertFormContainsNButtonsWithSizeClass(3, 7);
```

Comment:
7 buttons to display, and the max number of buttons for ui size 3 is 7, so there're all displayed (no "More"). Before, 6 were displayed, and one was in the dropdown